src/UiPath.CoreIpc.Tests/Implementation/SystemService.cs

```diff
@@ -175,5 +180,27 @@ public async Task<Stream> Echo(Stream input, CancellationToken cancellationToken
             result.Position = 0;
             return result;
         }
+
+        [DllImport("kernel32.dll", SetLastError = true)]
+        public static extern bool CancelIoEx(IntPtr handle, IntPtr lpOverlapped);
```

Comment:
can define it as a nested function (on develop)

Reply:
Now the CancelIoEx it's public and used also in PipeCancelIoOnClient test. Maybe, on develop, should be defined on different place (e.g. a separate class)